frontend/src/metabase/querying/components/FilterValuePicker/StaticValuePicker/StaticValuePicker.tsx

```diff
@@ -17,59 +16,24 @@ interface StaticValuePickerProps {
 export function StaticValuePicker({
   selectedValues,
   placeholder,
-  canAddValue,
+  shouldCreate,
   autoFocus,
   onChange,
   onFocus,
   onBlur,
 }: StaticValuePickerProps) {
-  const [lastValues, setLastValues] = useState(selectedValues);
-  const [isFocused, setIsFocused] = useState(false);
-  const visibleValues = isFocused ? lastValues : selectedValues;
-  const [searchValue, setSearchValue] = useState("");
-
-  const handleChange = (newValues: string[]) => {
-    setLastValues(newValues);
-    onChange(newValues);
-  };
-
-  const handleFocus = (event: FocusEvent<HTMLInputElement>) => {
-    setIsFocused(true);
-    setLastValues(selectedValues);
-    onFocus?.(event);
-  };
-
-  const handleBlur = (event: FocusEvent<HTMLInputElement>) => {
-    setIsFocused(false);
-    setLastValues(selectedValues);
-    setSearchValue("");
-    onBlur?.(event);
-  };
-
-  const handleSearchChange = (newSearchValue: string) => {
-    setSearchValue(newSearchValue);
-
-    const isValid = canAddValue(newSearchValue);
-    if (isValid) {
-      onChange?.([...lastValues, newSearchValue]);
-    } else {
-      onChange?.(lastValues);
-    }
-  };
-
   return (
-    <MultiSelect
-      data={visibleValues}
-      value={visibleValues}
-      searchValue={searchValue}
+    <MultiAutocomplete
+      data={[]}
```

Comment:
Is this correct?

Reply:
Yes - there are no predefined options in this case